components/storage/blockdevices/COMPONENT_SPIF/mbed_lib.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "spif-driver",
+    "config": {
+        "SPI_MOSI": "SPI_MOSI",
+        "SPI_MISO": "SPI_MISO",
+        "SPI_CLK":  "SPI_SCK",
+        "SPI_CS":   "SPI_PERSISTENT_MEM_CS",
```

Comment:
Why don't you use "SPI_CS":   "SPI_CS"  ?
In our PinNames.h, SPI_CS already exists....?

Reply:
While the SPI bus will be the same the CS may point to a different module than the storage.